code/modules/gear_presets/uscm_medical.dm

```diff
@@ -78,53 +86,31 @@
 
 	minimap_icon = list("medic" = MINIMAP_ICON_COLOR_DOCTOR)
 
-	utility_under = list(/obj/item/clothing/under/rank/medical/green)
-	utility_hat = list(/obj/item/clothing/head/surgery/green)
-	utility_gloves = list(/obj/item/clothing/gloves/latex)
-	utility_shoes = list(/obj/item/clothing/shoes/white)
-	utility_extra = list(/obj/item/clothing/suit/storage/labcoat)
-
-	service_under = list()
-	service_over = list()
-	service_hat = list()
-	service_shoes = list(/obj/item/clothing/shoes/laceup)
-
-	dress_under = list(/obj/item/clothing/under/suit_jacket)
-	dress_over = list()
-	dress_hat = list()
-	dress_gloves = list(/obj/item/clothing/gloves/marine/dress)
-	dress_shoes = list(/obj/item/clothing/shoes/laceup)
-
 /datum/equipment_preset/uscm_ship/uscm_medical/doctor/load_gear(mob/living/carbon/human/H)
 	var/backItem = /obj/item/storage/backpack/marine/satchel
 	if (H.client && H.client.prefs && (H.client.prefs.backbag == 1))
 		backItem = /obj/item/storage/backpack/marine
 
-	H.equip_to_slot_or_del(new /obj/item/device/radio/headset/almayer/doc(H), WEAR_L_EAR)
+	H.equip_to_slot_or_del(new backItem(H), WEAR_BACK)
```

Comment:
snake case back_item

Reply:
> Needs to spawn with appropriate headset.
> 
> /obj/item/clothing/under/rank/medical needs removed
> 
> Remove reagent scanner HUD.

1-for the headset i double check and all the medical can get their headset from their vendor.
do you want me to reverse that and make them spawn with their headset anyway?
all headset are the same for nurse research and doc./obj/item/device/radio/headset/almayer/doc
 
2- unsure what you mean by /obj/item/clothing/under/rank/medical need to be removed as a lot of items are called like that.

3-do you want me to remove the reagent scanner HUD from all the vendors?

Reply:
what you want me to do with H.equip_to_slot_or_del(new backItem(H), WEAR_BACK)?
do you want me to delete it? i honestly don't see what this line is doing....
look like some sort of check to put items in the backpack...

Reply:
it as been done